site/docs/v1/tech/installation-guide/kubernetes/fusionauth-deployment.adoc

```diff
@@ -176,9 +176,17 @@ Consult the setup guides for instructions for AWS, GCP, and more.
 * `database.root.user`
 * `database.root.password`
 
+If you have not setup a Postgres user for FusionAuth you may additionally need to use the root user credentials for these so FusionAuth can configure the database.
```

Comment:
```suggestion
If you have not set up a postgres username and password for FusionAuth, you can root user credentials for these values so FusionAuth can configure the database.
```

Hmmm. Something isn't right here. FusionAuth should use the root credentials and then create a postgres user. Is that not what happened?

Reply:
So in the initial install of the helm chart you have to declare the root.user credentials and the user credentials or it won't configure the database. So you have to provide the same credentials, or I assumed if you created a non root user for FusionAuth you could put that there? But database.user, and database.password are required but not called out in the doc what it should be.

Let me create a second user in postgres and run it and see what occurs. I made assumptions here without testing thinking they could be different if the user was created ahead of instaling the chart.

Reply:
Hi @mooreds, I tested this and when you install the bitnami/postgres chart there is only the root user. You can create an additional super user and then use those credentials for FusionAuth so that FusionAuth does not have root access.

I have tested this and happy to walk you through if you like